pyproject.toml

```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 package-mode = false
 
 [tool.poetry.dependencies]
-python = "^3.11"
```

Comment:
Coveralls defined that it doesnt support Python 3.13 (`<3.13`) and since `^3.11` does support it it prevented from upgrade